spec/services/usps_in_person_proofing/enrollment_helper_spec.rb

```diff
@@ -179,10 +181,12 @@
         end
       end
 
-      it 'sets enrollment status to pending and sets established at date and unique id' do
+      it 'sets enrollment status to pending, sponsor_id to usps_ipp_sponsor_id,
+       and sets established at date and unique id' do
```

Comment:
adding a newline inside this string will put a bunch of spaces in the test name, no? might be easier to use heredoc literal here
```suggestion
      it <<~STR.squish do
        sets enrollment status to pending, sponsor_id to usps_ipp_sponsor_id,
        and sets established at date and unique id
      STR
```
```suggestion
      it 'sets enrollment status to pending, sponsor_id to usps_ipp_sponsor_id,
       and sets established at date and unique id' do
```

Reply:
You are correct, it adds a bunch of spaces in the test name. Resolved with commit c553ff7ffcc68ae8970f3f07d57aa9029612ee2c